AMPFloatingTextField/Classes/AMPFloatingTextField.swift

```diff
@@ -29,6 +34,11 @@ open class AMPFloatingTextField: UITextField { // swiftlint:disable:this type_bo
     // MARK: Colors
     fileprivate var cachedTextColor: UIColor?
     private let borderLayer = CALayer()
+    var rules: [Rule]?
+    fileprivate var isTextValid: Bool = false
+    fileprivate var cachedErrorMessage: String!
+    
+    open var isImmediateValidation: Bool = false
```

Comment:
can modify to this : isImmediateValidation to isResignResponderValidationRequired